src/calculateRentalCost.js

```diff
@@ -4,7 +4,44 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const discountUpToSix = 20;
+  const discountSevenMore = 50;
+
+  if (days <= 0) {
+    return 'You need to rent for at least one day';
```

Comment:
The function should return a number representing the total cost. Returning a string in case of an invalid input is inconsistent. Consider throwing an error or returning a special numeric value (like 0) that indicates an invalid input.